contracts/account/AccountCore.sol

```diff
@@ -95,14 +95,15 @@ abstract contract AccountCore is AbstractSigner, IAccount {
     }
 
     /**
-     * @dev Virtual function that returns the signable hash for a user operations. Some implementation may return
-     * `userOpHash` while other may prefer a signer-friendly value such as an EIP-712 hash describing the `userOp`
-     * details.
+     * @dev Virtual function that returns the signable hash for a user operations. Since v0.8.0 of the entrypoint,
+     * `userOpHash` is a EIP-712 hash that can be signed directly.
```

Comment:
```suggestion
     * `userOpHash` is an EIP-712 hash that can be signed directly.
```

Reply:
AFAIK, we say "a hash" and not "an hash"